<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterLasWizardPage</class>
 <widget class="QWizardPage" name="ParameterLasWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="upperGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="newParameterSetLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="existingParameterSetRadioButton">
        <property name="text">
         <string>Existing parameter set</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="existingParameterSetComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="newParameterSetRadioButton">
        <property name="text">
         <string>New parameter set</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="parametersTableWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="lowerGroupbox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="viewfilePushButton">
        <property name="text">
         <string>View file </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="lowerHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
